lib/devices/android/android-hybrid.js

```diff
@@ -255,4 +287,38 @@ androidHybrid.initAutoWebview = function (cb) {
   }
 };
 
+// get the correct chromedriver executable path based on our system
+// TODO: don't download/build chromedriver in reset.sh, instead let this be
+// something that the appium-chromedriver package manages
+androidHybrid.initChromedriverPath = function (cb) {
+  if (this.args.chromedriverExecutable) {
```

Comment:
just responding to this now: the appium-chromedriver package ensures it's a valid path